prisma/migrations/20241126204944_added_created_at_in_link_table/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE `link` ADD COLUMN `createdAt` DATETIME(3) NOT NULL DEFAULT CURRENT_TIMESTAMP(3);
```

Comment:
**issue**: Ensure that the `createdAt` column is indexed if it will be frequently queried, especially for deletion of expired links.